pkg/logconfig/config.go

```diff
@@ -31,3 +36,22 @@ const (
 	// Webhook is the name of the override key used inside of the logging config for Webhook Controller.
 	Webhook = "webhook"
 )
+
+func ConfigName() string {
+	if ns := os.Getenv(ConfigMapNameEnv); ns != "" {
```

Comment:
when I see ns, I think namespace. nit, consider changing this to something like: cm

Reply:
Done